routes/signout.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+
+router.post("/", (req, res) => {
+  res.clearCookie("rds-session");
+  res.redirect("/healthcheck");
```

Comment:
Why are we redirecting to `/healthcheck`?

Reply:
Nothing specific main task was done so I preferred to redirect. Any suggestions for what could be done? instead of redirecting?

Reply:
Read what we are doing in other endpoints where we modify something. That'll be helpful

Reply:
ok

Reply:
Also, what should happen in the case where user is already logged out?

Reply:
As per the latest commit I resolved this issue please check it and let me know if it can be resolved